### PR TITLE
update bm upi for DRtest1

### PR DESCRIPTION
**Pull request by geliu2016**

### CODE REVIEW

**Comment by geliu2016** [comment]
@kuiwang02 could you please take a reivew? thanks

**Comment by geliu2016** [comment]
@kuiwang02 could you please take a reivew? thanks

**Comment by kuiwang02** [comment]
@geliu2016 
1, need to run one test to see if it is OK to create cluster. 
2, please pay attention to the bm resource usage.

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by kuiwang02** [comment]
I just start test for you. please monitor if it create cluster successfully.
if not, please query worflow owner to check it.

after it create cluster successfully, please ping me to review it again.

**Comment by geliu2016** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by geliu2016** [comment]
@kuiwang02 is it ok, now? thanks

**Comment by kuiwang02** [comment]
> @kuiwang02 is it ok, now? thanks

@geliu2016 please refer to https://github.com/openshift/release/pull/37786#issuecomment-1489730704 to check if it is ok.

now I do not see the cluster is created successfully.
so, need to trigger it

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by kuiwang02** [comment]
I just start [test](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/37786/rehearse-37786-pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi/1643146049927254016) again.  please monitor if the cluster is created successfully.

**Comment by geliu2016** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by geliu2016** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by geliu2016** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by aleskandro** [comment]
/assign

(just to keep in my profile's shortcuts)

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml [hunk 1]
```diff
@@ -179,6 +179,21 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: debug-disasterrecovery-baremetal-upi
+  optional: true
+  run_if_changed: test/extended/disaster_recovery/
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
```
Comment (aleskandro) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml:186]: To add under the steps stanza:
```
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```

**Comment by aleskandro** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by aleskandro** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-disasterrecovery-baremetal-upi

**Comment by geliu2016** [comment]
@kuiwang02 The bm upi cluster setup successfully, could you please take a view? thanks

**Comment by kuiwang02** [comment]
> @kuiwang02 The bm upi cluster setup successfully, could you please take a view? thanks

@geliu2016 just double confirm with you: the cluster you create is based on arm, not amd. is it what you want?
thanks

**Comment by geliu2016** [comment]
> not
@kuiwang02 I discussed with [aleskandro](https://github.com/aleskandro), curretnly, amd64 still have some issues, and arm64 is ready, so @gangwgr dose arm64 could support your test case also?

**Comment by kuiwang02** [comment]
@geliu2016 ok, got it. thanks
so, I approve the PR now?

**Comment by geliu2016** [comment]
> @geliu2016 ok, got it. thanks so, I approve the PR now?

@kuiwang02 Yes, I just have a quick sync with rahul, he reply me lgtm with arm64 cluster.

**Pull request merged**